### PR TITLE
Feat/13 define and create categories in the database

### PR DESCRIPTION
**Pull request by Valsuh45**

### CODE REVIEW

File: prisma/schema.prisma [hunk 1]
```diff
@@ -72,6 +81,11 @@ model Product {
     images   Image[]          @relation("ProductImages")
     variants ProductVariant[]
 
+    createdBy   User   @relation(fields: [createdById], references: [id])
+    createdById String
```
Comment (stephane-segning) [inline prisma/schema.prisma:85]: Why do we need a this for a product?

File: src/server/api/root.ts [hunk 2]
```diff
@@ -8,14 +9,15 @@ import { createCallerFactory, createTRPCRouter } from '@cps/server/api/trpc';
...
+  category: categoryRouter, // Add the category router
 });
 
 // export type definition of API
 export type AppRouter = typeof appRouter;
 
 /**
  * Create a server-side caller for the tRPC API.
- * @example
+ * @exampleroot.ts
```
Comment (stephane-segning) [inline src/server/api/root.ts:20]: ???

File: prisma/schema.prisma (hunk 1 above)
Comment (Valsuh45) [reply prisma/schema.prisma]: It was just an additional parameter and will be removed

**Review by stephane-segning — APPROVED** [review]

**Review by mbunwe-victor — APPROVED** [review]
Nice job @Valsuh45

**Pull request merged**